.github/workflows/comment-pr-on-check-failure.yml

```diff
@@ -10,34 +10,16 @@ on:  # yamllint disable-line rule:truthy
       - completed
 
 jobs:
-  on-failure:
+  comment_pr:
     runs-on: ubuntu-latest
-    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
     env:
       CHECK_GUIDELINE: "[Guide to the different CI checks and resolution guidelines](https://docs.magmacore.org/docs/next/contributing/contribute_ci_checks)"
+      WORKFLOW_NAME: "${{ github.event.workflow.name }}"
+      WORKFLOW_STATUS: "${{ github.event.workflow_run.conclusion }}"
+      COMMIT_ID: "${{ github.event.workflow_run.pull_requests[0].head.sha }}"
+      COMMIT_ID_URL: "${{ github.event.repository.html_url }}/commit/${{ github.event.workflow_run.pull_requests[0].head.sha }}"
     steps:
       # Retrieve PR number from triggering workflow artifacts
-      - name: 'Download artifact'
```

Comment:
If we don't need this part, can you delete the part where we upload the number as an artifact in the other workflows.

Reply:
> If we don't need this part, can you delete the part where we upload the number as an artifact in the other workflows.

@quentinDERORY - I will take care of this change in other workflows in next version. I need to analyze more on those other workflows to ensure this change will not break anything.

Reply:
I can ensure it won't break anything if this workflow is working.

Reply:
> I can ensure it won't break anything if this workflow is working. And if it breaks something, it will be better to revert this whole change.

Hehehe. Made the changes and pushed. Please review @quentinDERORY